pandora-common/src/space/space.ts

```diff
@@ -83,6 +84,8 @@ export const SpaceDirectoryConfigSchema = SpaceBaseInfoSchema.extend({
 		shardId: z.string().optional(),
 		/** Automatically grants admin to every developer on enter */
 		autoAdmin: z.boolean().optional(),
+		/** Disable safemode cooldown for everyone inside the space */
+		disableCooldown: z.boolean().optional(),
```

Comment:
Nitpick: Maybe a bit more explicit name `disableSafemodeCooldown`, in case we ever get other cooldowns (which is likely, for example apply/modify/remove item cooldown)